tutorials/cosmwasm.md

```diff
@@ -246,6 +256,18 @@ This will place the optimized Wasm bytecode at `artifacts/cw_nameservice.wasm`.
 
 Let's now deploy our smart contract!
 
+We will need to do this in the docker container that the CosmWasm rollup is running. So first let's move the compiled contract to the container:
+
+```bash
+docker cp artifacts/cw_nameservice.wasm wasm--c71b0308616d40ad919ad24c3d14f35b:/root/cw_nameservice.wasm
```

Comment:
We should add instructions for ARM64 as well? Since we're copying it as `cw_nameservice.wasm` into the docker container? ARM64 machines expect a file called `cw_nameservice-aarch64.wasm` instead

Reply:
removed need for machine specific commands with another docker image.